gslib/commands/signurl.py

```diff
@@ -162,6 +162,13 @@
                Note that both options have a maximum allowed duration of
                12 hours for a valid link.
 
+  -b <project> Allows you to specify a user project that will be billed for
+               requests on the signed URL. This is useful for generating
```

Comment:
Suggestion: Change "on the" to "that use"

Reply:
Done